package.json

```diff
@@ -157,6 +157,7 @@
     "react": "18.2.0",
     "react-beautiful-dnd": "13.1.1",
     "react-dom": "18.2.0",
+    "react-fast-compare": "3.2.2",
```

Comment:
leaving a reminder to delete as per your comment in the other PR that this is not used